docs/notes/2.24.x.md

```diff
@@ -50,6 +50,8 @@ Fixed a coursier fetch wrapper script bug on nixos.
 
 Fixed a jvm.sh script bug on nixos.
 
+Lockfiles are now deterministic as the dependency cooridates to be resolved are sorted before being passed to couriser.
```

Comment:
```suggestion
Lockfiles are now deterministic as the dependency coordinates to be resolved are sorted before being passed to couriser.
```

Reply:
Thanks! Fixed the failed lint as well in a force-push.